src/goose.rs

```diff
@@ -811,21 +811,22 @@ impl GooseRequestCadence {
     }
 }
 
+pub trait GooseUserData: Downcast + Send + Sync + 'static {}
```

Comment:
This needs documentation (can be in followup).

Reply:
(As it's a pub trait, documentation is important and will show up at https://docs.rs/goose/)